src/bp/core/routers/bots/index.ts

```diff
@@ -30,6 +30,7 @@ import { URL } from 'url'
 import { disableForModule } from '../conditionalMiddleware'
 import { CustomRouter } from '../customRouter'
 import { checkTokenHeader, needPermissions } from '../util'
+import { fileSync } from 'tmp'
```

Comment:
single import with no other changes in that file, maybe an auto import?

Reply:
yeah auto import